common/azure/src/main/kotlin/com/cosmotech/api/azure/adx/AzureDataExplorerClient.kt

```diff
@@ -82,6 +83,29 @@ class AzureDataExplorerClient(private val csmPlatformProperties: CsmPlatformProp
         }
     return healthBuilder.withDetail("baseUri", baseUri).build()
   }
+
+  public fun deleteDataFromScenarioRunId(
```

Comment:
Kotlin methods and functions are `public` by default.
```suggestion
  fun deleteDataFromScenarioRunId(
```